10.0/apache/docker-entrypoint.sh

```diff
@@ -1,9 +1,45 @@
 #!/bin/bash
 set -e
 
-if [ ! -e '/var/www/html/version.php' ]; then
-    tar cf - --one-file-system -C /usr/src/nextcloud . | tar xf -
+# version_greater A B returns whether A > B
+function version_greater() {
+	[[ "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1" ]];
```

Comment:
I just tried this on an alpine box, and the busybox sort is complaining about the `V`...

I ended up using this:
https://github.com/pierreozoux/docker-alpine/blob/03573c088fc71bc9bd6a6e0b4af262935289621c/builder/scripts/apk-install#L4-L6